examples/coap/fauxhoc.py

```diff
@@ -96,13 +148,14 @@ async def main():
 
     priv, pub = lakers.p256_generate_key_pair()
 
-    c_i = 0x08
+    # We only run one connection so we don't care, but let's spread it
+    c_i = bytes([random.randint(0, 23)])
     initiator = lakers.EdhocInitiator()
-    message_1 = initiator.prepare_message_1(c_i=c_i)
+    message_1 = initiator.prepare_message_1(c_i)
 
     msg1 = Message(
```

Comment:
Diagnostic output has been enhanced.

I've kept `c_i=c_i` because I plan to propose more explicit argument naming in Lakers (there are too many byte-ish typed items that are easily confused), and Python arguments don't have the nice Rust struct feature of implicit naming.